OpenRA.Mods.Common/Pathfinder/PathSearch.cs

```diff
@@ -100,16 +100,27 @@ protected override void AddInitialCell(CPos location)
 		/// This function analyzes the neighbors of the most promising node in the Pathfinding graph
 		/// using the A* algorithm (A-star) and returns that node
 		/// </summary>
-		/// <returns>The most promising node of the iteration</returns>
-		public override CPos Expand()
+		/// <returns>The most promising node of the iteration or false if expansion is no longer possible</returns>
```

Comment:
These are no longer quite correct. I suggest something like:
```csharp
/// <summary>
/// This function analyzes the neighbors of the most promising node in the Pathfinding graph
/// using the A* algorithm (A-star).
/// </summary>
/// <param name="mostPromisingNode">If the graph can be expanded, contains the next lowest cost cell.</param>
/// <returns>True if the graph was expanded and mostPromisingNode contains a valid cell, otherwise false.</returns>
```